elasticlib/Elastic.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot;
+
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.PubSubOption;
+import edu.wpi.first.networktables.PubSubOptions;
+import edu.wpi.first.networktables.StringArrayEntry;
+import edu.wpi.first.networktables.StringArrayPublisher;
+import edu.wpi.first.networktables.StringArrayTopic;
+
+public final class Elastic {
+    private static final StringArrayTopic topic = NetworkTableInstance.getDefault().getStringArrayTopic("notifications");
```

Comment:
Maybe change the topic to something like "elastic/notifications" in case if there's ever some library that also publishes to a topic called notifications?

Reply:
Thinking of renaming it to "elastic/alerts" since I think that would make more sense.

Robot alerts vs Elastic Notifications. Maybe an icon on the notification can differentiate the two?